ray_julia_jll/test/reference_counting.jl

```diff
@@ -0,0 +1,26 @@
+@testset "Reference counting" begin
+    using ray_julia_jll: FromRandom, ObjectID, Hex, AddLocalReference, RemoveLocalReference,
+                         GetAllReferenceCounts, _keys, _getindex, GetCoreWorker
+
+    worker = GetCoreWorker()
+    # need to convert to hex string because these are just pointers
+    has_count(oid) = Hex(oid) in Hex.(_keys(GetAllReferenceCounts(worker)))
```

Comment:
Is this still true given that `==` was implemented?

Reply:
Can this be simplified to this now?
```suggestion
    # need to convert to hex string because these are just pointers
    has_count(oid) = oid in _keys(GetAllReferenceCounts(worker))
```

Reply:
let's give it a try...

Reply:
nope, teh `==` fix is in #138